models/user.js

```diff
@@ -8,6 +9,11 @@ const schema = new mongoose.Schema({
   lastName: { type: String, default: '' },
   username: { type: String, required: true, unique: true },
   password: { type: String, required: true },
+  list: [{
+    item: { type: String },
+    aisle: { type: String },
+    picked: { type: Boolean}
```

Comment:
checked may work better here, just to keep wording consistent

Reply:
+1
Also, 'aisle' will likely transition to become a "category" or something similar, but for now a String will be fine